codespell_lib/data/dictionary.txt

```diff
@@ -10074,6 +10074,8 @@ dnamically->dynamically
 dne->done
 do'nt->don't
 doalog->dialog
+doamin->domain
```

Comment:
Suggest dopamine too

Reply:
I added `dopamine` suggestion for both `doamin` and `doamine`

Reply:
Thanks, I suspect the former is less likely, but it can't hurt.

Reply:
Ignore that, I get what you mean now.